docs/edgeql/index.rst

```diff
@@ -1,104 +1,85 @@
-(but often convenient) cousin of SDL. DDL is used to transform schema
-step by step instead of all at once. The order is important in DDL
-because it involves issuing one command after another instead of
-describing the final form.
-
-So you can't create ``Movie`` first using DDL because it links
-to ``Person``.  The two are created in this order:
-
-.. code-block:: edgeql-repl
-
-    db> CREATE TYPE Person {
-    ...    CREATE REQUIRED PROPERTY name -> str;
-    ... };
-    CREATE: OK
-    db> CREATE TYPE Movie {
-    ...     CREATE REQUIRED PROPERTY title -> str;
-    ...     CREATE REQUIRED LINK director -> Person;
-    ...     CREATE MULTI LINK actors -> Person;
-    ... };
-    CREATE: OK
-
-SDL is sort of like a 3D printer: you set the final shape and it puts
-it together for you. DDL is like building a house with traditional
-methods: to add a window you first need a frame, to have a frame you
-need a wall, and so on. But DDL is great for making quick changes to
-your schema without a new migration, in the same way that you can
-replace a window without describing the whole house to do it.
-In practice, most people stick to SDL until they get comfortable
-and only then begin to experiment with DDL.
-
-**The remainder:** this includes infrequently used but critical items
-involved in database administration. Start with the
-**Administration** link below to see how to configure a database,
-set roles, passwords, and so on using EdgeDB.
-
-
 .. toctree::
     :maxdepth: 3
     :hidden:
 
-    concepts
-    expressions/index
-    commands/index
+    literals
+    sets
+    paths
     types
-    sdl/index
-    ddl/index
-    introspection/index
-    admin/index
-    lexical
-    output
-    eval
+    funcops
+    parameters
+    select
+    insert
+    update
+    delete
+    with
+    for
+    transactions
+
+EdgeQL is a next-generation query language designed to match SQL in power and
+surpass it in terms of clarity, brevity, and intuitiveness. It's used to query
+the database, insert/update/delete data, modify/introspect the schema, manage
+transactions, and more.
+
+Design goals
+------------
+
+EdgeQL is a spiritual successor to SQL designed with a few core principles in
+mind.
+
+**Compatible with modern languages**. A jaw-dropping amount of effort has been
+spent attempting to `bridge the gap <https://en.wikipedia.org/wiki/
+Object%E2%80%93relational_impedance_mismatch>`_ between the *relational*
+paradigm of SQL and the *object-oriented* nature of modern programming
+languages. EdgeDB sidesteps this problem by modeling data in an
+*object-relational* way.
+
+**Strongly typed**. EdgeQL is *inextricably tied* to EdgeDB's rigorous
+object-oriented type system. The type of all expressions is statically
+inferred by EdgeDB.
+
+**Easy to learn**. Postgres-flavored SQL contains `469 keywords
+<https://www.postgresql.org/docs/current/sql-keywords-appendix.html>`_. By
+comparison EdgeQL contains 80.
```

Comment:
I don't think this is a fair point. We do have a limited and comparatively small set of reserved keywords, but we have a lot of unreserved and future-reserved ones. If you count them all then it's 470 vs 180 which isn't that much big of a difference (compared to say Python having like 20 keywords).

I'd just say something along the lines of "we payed a great deal of attention to make sure that edgeql is linear (no "postfix" operators and always reads top to bottom) and high-level (no need to bother yourself with joins).

Reply:
I'll update to 180. Even 469 vs 180 sounds impressive. This point was made in We Can Do Better Than SQL and I think it's quite compelling.

Reply:
I'm not convinced. It would be compelling if we surpassed SQL in all aspects but in reality we're still catching up; this isn't a fair comparison. I really don't like the keyword counting theme.

Reply:
I think it's important to distinguish Query keywords from DDL keywords.  The majority of our keywords are in DDL (which you would rarely need to write).  The QL part is actually quite small.  This is less true for SQL.

Reply:
Also, future reserved keywords don't count, since the point is about the _current_ size of the language that you need to learn to be productive.

Reply:
@elprans wanna suggest alternative wording?